debezium-core/src/main/java/io/debezium/jdbc/JdbcValueConverters.java

```diff
@@ -1149,27 +1149,6 @@ protected Object convertBits(Column column, Field fieldDefn, Object data, int nu
                 Boolean value = (Boolean) data;
                 r.deliver(new byte[]{ value.booleanValue() ? (byte) 1 : (byte) 0 });
             }
-            else if (data instanceof Short) {
```

Comment:
Related to that point in the description:
- removed legacy format handling fot BIT related types in JdbcValueConverters#convertBits which were maybe leftovers from PR #1408?

I think these cases were leftovers from the past?

Reply:
I'm not totally sure tbh. It could be some drivers/driver versions deliver those types. Personally, I'd leave it as-is. If you feel strongly about it, we can remove it and see whether some users complain.

Reply:
at least no test fails :man_shrugging:

Reply:
True that. Let's go for the removal.